feat: reformat outputs to always show if the secret is known (#505)
GitGuardian/ggshield: changelog.d/20230404_144426_alina.tuholukova_add_known_secrets_url.md

<!--
A new scriv changelog fragment.

Uncomment the section that is right (remove the HTML comment wrapper).
-->

<!--
### Removed

- A bullet item for the Removed category.

-->
<!--
### Added

- A bullet item for the Added category.

-->

### Changed

> - The output of the scan command includes now the information if the secret is known by user's GitGuardian
>   dashboard. If yes, the output also includes the incident url.

Reviewer: I think this can be simplified to:

```suggestion
- `ggshield secret scan` output now includes a link to the incident if the secret is already known on the user's GitGuardian dashboard.
```
